### ISSUE TITLE
When multiple lists have same key name, need more elaborate error message in case of configuration having duplicate keys

### ISSUE THREAD

**Issue by DeekshaBhandary** [clicon/clixon#362]
```
list l1{
key "name";
leaf name {
 type string;
}
}

list l2{
key "name";
leaf name {
 type string;
}
}
```
**json config file:**
```
{
        "ietf-restconf:data": {
                "config:l1": [
                        {
                                 "name" : "test1"
                        },
                        {
                                 "name" : "test1"
                        }
                ],
                "config:instance": [
                        {
                                "name" : "test2"
                        },
                        {
                                 "name" : "test3"
                        }
                ]
        }
}
```
Loading this file throws error with just the node name that has uniqueness issue. In case of huge configs it is difficult to find the exact
config causing the issue. Having node's xpath or parent info in the error message can be useful

```
cli> load config test.json
Sep  9 10:48:26: Editing configuration: application operation-failed <non-unique>name</non-unique>: data-not-unique 
cli>
```

**Comment by olofhagsand** [comment]
Agreed.
Although not technically a "bug" it is a usability issue. Please continue to report such observations.

**Comment by olofhagsand** [comment]
Fixed by conforming to RFC 7950 Section 15.1 replacing name with instance-identifier. 
Also adding correct namespace to the non-unique tag.
Please verify.
